### ISSUE TITLE
Help text: fix a11y and props

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#694]
By default, the help text uses a gray color `$color_grey_medium_dark` set to `#888888`. The contrast ratio against a white background is 3.54:1 which is acceptable just for "large text" (24 pixels or 18.5 pixels bold).

For normally sized text, the contrast ratio should be at least 4.5:1 so the lightest gray that can be used is `$color_grey_text` set to `#646464`.

- the prop `textColor` is OK but the CSS property should be `color`, not `textColor`
- the default font-size is `0.8em` which computes to 12.8 pixels in the standalone app but in the plugin it's only 10.4 pixels because the WordPress uses 13 pixels as base font size

**Comment by afercia** [comment]
To evaluate: as discussed with @hedgefield the introduction of a new gray e.g. `#767676` as the lightest one to use for text against a white background (contrast 4.54:1).

**Comment by afercia** [comment]
Also, in a few places we're using custom styled components for some help text, for example in the SynonymsInput:
```
const ExplanationText = styled.p`
	font-size: .8em;
`;
```

or directly in the plugin SeoAnalysis.js
```
const ExplanationText = styled.p`
`;
```

Seems to me it would be better to standardize and always use the `HelpText` component. Also, not sure why the help text is passed to HelpText as a prop, and not using `children`.
